requirements.lock

```diff
@@ -78,6 +78,7 @@ raven==6.10.0
 requests==2.22.0
 requests-oauthlib==1.2.0
 rsa==4.0
+rules==2.0.1
```

Comment:
Ah OK, this is where `add_perm()`, etc...comes from, and this is what Ravi meant by "there is no database":

> rules is a tiny but powerful app providing object-level permissions to Django, without requiring a database. [...]